content/general/concepts/software-development-life-cycle/terms/v/v.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'V'
+Description: 'A model of software development that was based on the Waterfall model.'
+Subjects:
+  - 'Mobile Development'
+  - 'Web Development'
+Tags:
+  - 'Development'
+  - 'Models'
+CatalogContent:
+  - 'learn-how-to-code'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **V model** is a Software Development Life Cycle (SDLC)  that is based on the [Waterfall model](https://www.codecademy.com/resources/docs/general/software-development-life-cycle/waterfall) and emphasizes the Verification and Validation of the product. Each phase of this model is operated along with a level of testing.
+
+The V model was first put into use in the late twentieth century and has been widely used ever since. It can be divided in five main phases:
```

Comment:
```suggestion
The V-model was first implemented in the late twentieth century and has been widely used ever since. It can be divided in five main phases:
```